### ISSUE TITLE
Different number of ground true instances and predictions in eval_ucf101.py

### ISSUE THREAD

**Issue by MinLiAmoy** [kenshohara/3D-ResNets-PyTorch#139]
Dear @kenshohara,

I tried to evaluate the pretrained resnet18 on UCF101 split 1 using eval_ucf101.py.
The top-1 error is 0.09886 which makes sense. But the number of ground true instances and predictions are always not the same value. e.g.

> [INIT] Loaded annotations from validation subset.
	Number of ground truth instances: 3783
	Number of predictions: 37820
 
I checked the codes and think it should be consistent. Did I miss something?
Thanks in advance!

**Comment by 1483852602** [comment]
I have the same question with you ...

[INIT] Loaded annotations from validation subset.
        Number of ground truth instances: 3783
        Number of predictions: 37820
[RESULTS] Performance on ActivityNet untrimmed video classification task.
        Error@1: 0.9883690192968544

**Comment by MinLiAmoy** [comment]
@1483852602 Hi, 
from function **calculate_video_results** in test.py, we store top-10 labels and results. It seems and the number of predictions will return the number of results,  but no the number of videos.

**Comment by Ixiaohuihuihui** [comment]
@MinLiAmoy  I met the same error.
Then I change the code:
`if __name__ == "__main__":
    hmdb = HMDBclassification('/data/Datasets/dailh/Datasets/HMDB/hmdb51_1.json',
                                  '/data/Datasets/dailh/Datasets/HMDB/results/val.json',subset='validation', top_k=1)
    hmdb.evaluate()
    print(hmdb.hit_at_k)`

I add :  subset='validation'
the error has been solved.

**Comment by ZHUXUHAN** [comment]
> @MinLiAmoy I met the same error.
> Then I change the code:
> `if __name__ == "__main__": hmdb = HMDBclassification('/data/Datasets/dailh/Datasets/HMDB/hmdb51_1.json', '/data/Datasets/dailh/Datasets/HMDB/results/val.json',subset='validation', top_k=1) hmdb.evaluate() print(hmdb.hit_at_k)`
> 
> I add : subset='validation'
> the error has been solved.

hello, I just train the model, but  I just did not  get the val.json file ,can you tell me how to get the file?
